pkg/sechttp/httpclient.go

```diff
@@ -41,17 +41,17 @@ func DispatchHTTPRequest(httpClient utils.HTTPClient, originalRequest *http.Requ
 
 	// Should be a 401 (bearer only) but is infact a 302 (Redirect to a login page)
 	keycloakLoginErrorStatus := http.StatusFound
-	logr.Tracef("Getting Connection: %v\n", connectionID)
+	logr.Tracef("Getting Connection: %v\n", conInfo.ID)
 
 	// Get the remote connection details
-	con, conErr := connections.GetConnectionByID(connectionID)
+	con, conErr := connections.GetConnectionByID(conInfo.ID)
 	if conErr != nil {
 		return nil, &HTTPSecError{errOpNoConnection, conErr.Err, conErr.Desc}
 	}
 
 	// Get the current access token from the keychain
 	logr.Traceln("Retrieving an access token from the keychain")
-	conID := strings.TrimSpace(strings.ToLower(connectionID))
+	conID := strings.TrimSpace(strings.ToLower(conInfo.ID))
```

Comment:
Isn't this redundant now (apart from the log message)-  shouldn't we be able to use the connection directly ?